spec/dummy/db/schema.rb

```diff
@@ -301,7 +301,7 @@
     t.string   "taggable_type"
     t.integer  "tagger_id"
     t.string   "tagger_type"
-    t.string   "context"
+    t.string   "context",       limit: 128
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.